pages/intentpreview.py

```diff
@@ -13,110 +13,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# from google.appengine.api import users
 from api.converters import feature_entry_to_json_verbose
-
-from internals import core_enums
-from internals import processes
+from api.intents_api import compute_subject_prefix
+from internals.core_enums import INTENT_DRAFT_TYPES_BY_STAGE_TYPE, IntentDraftType
 from internals import stage_helpers
-from internals.core_models import Stage
+from internals.core_models import FeatureEntry, Stage
 from internals.review_models import Gate
 from framework import basehandlers
-from framework import permissions
 
-INTENT_PARAM = 'intent'
-LAUNCH_PARAM = 'launch'
 VIEW_FEATURE_URL = '/feature'
 
-
-def compute_subject_prefix(feature, intent_stage):
-  """Return part of the subject line for an intent email."""
-
-  if intent_stage == core_enums.INTENT_IMPLEMENT:
-    if feature.feature_type == core_enums.FEATURE_TYPE_DEPRECATION_ID:
-      return 'Intent to Deprecate and Remove'
-    else:
-      return 'Intent to Prototype'
-  elif intent_stage == core_enums.INTENT_EXPERIMENT:
-    return 'Ready for Developer Testing'
-  elif intent_stage == core_enums.INTENT_ORIGIN_TRIAL:
-    if feature.feature_type == core_enums.FEATURE_TYPE_DEPRECATION_ID:
-      return 'Request for Deprecation Trial'
-    else:
-      return 'Intent to Experiment'
-  elif intent_stage == core_enums.INTENT_EXTEND_ORIGIN_TRIAL:
-    if feature.feature_type == core_enums.FEATURE_TYPE_DEPRECATION_ID:
-      return 'Intent to Extend Deprecation Trial'
-    else:
-      return 'Intent to Extend Experiment'
-  elif intent_stage == core_enums.INTENT_SHIP:
-    if feature.feature_type == core_enums.FEATURE_TYPE_CODE_CHANGE_ID:
-      return 'Web-Facing Change PSA'
-    else:
-      return 'Intent to Ship'
-  elif intent_stage == core_enums.INTENT_REMOVED:
-    return 'Intent to Extend Deprecation Trial'
-
-  return f'Intent stage "{core_enums.INTENT_STAGES[intent_stage]}"'
-
-
 class IntentEmailPreviewHandler(basehandlers.FlaskHandler):
-  """Show a preview of an intent email, as appropriate to the feature stage."""
+  """Show a preview of an intent email. Used for testing the intent template."""
```

Comment:
"testing" sounds like unit testing, but I can imagine that "testing" here might meant that the user is trying something out.  If it is really just unit testing, it would be more clear to say "only for unit testing".